### ISSUE TITLE
make-directory error on linix

### ISSUE THREAD

**Issue by bennn** [tnelson/Forge#82]
from Piazza [`@119`](https://piazza.com/class/kjvxjqd2x1m490?cid=119):

```
make-directory: cannot make directory
  path: /Users/sarah/.config/forge/
  system error: Permission denied; errno=13
```

Lets do two things:
1. try to set permissions on the directory that `basedir` gives in `forge/logging/logging.rkt`
2. catch the exception, print a warning, and turn off logging if it happens

**Comment by bennn** [comment]
lol I don't know what I was thinking with this issue name

**Comment by bennn** [comment]
fixed in #81
